bokun.yaml

```diff
@@ -5537,413 +5734,535 @@ definitions:
+      inventorySupportsPricing:
         type: boolean
-        description: 'Whether ticket is generated per person or per booking.'
-      supportedAccessibilityTypes:
-        type: array
-        description: 'Specifies whether the activity is suitable for individuals with accessibility issues.'
-        items:
-          $ref: '#/definitions/AccessibilityType'
-      startPoints:
-        type: array
-        description: 'Only relevant if meetingType == MEET_ON_LOCATION or MEET_ON_LOCATION_OR_PICK_UP. List of all places where this activity can start or be joined.'
-        items:
-          $ref: '#/definitions/Place'
-      guidanceTypes:
-        type: array
-        description: 'The list of types of guidance on offer, along with what languages each is offered in.'
-        items:
-          $ref: '#/definitions/Guidance'
-      activityCategories:
-        type: array
-        description: 'The list of categories that the tour/activity/attraction falls into.'
-        items:
-          $ref: '#/definitions/ActivityCategory'
-      activityAttributes:
-        type: array
-        description: 'Attributes that describe the tour/activity/attraction.'
-        items:
-          $ref: '#/definitions/ActivityAttribute'
-      actualVendor:
+        description: 'If this activity is external inventory service product, then equals to SUPPORTS_PRICING of the connected plugin.'
+      inventorySupportsAvailability:
+        type: boolean
+        description: 'If this activity is external inventory service product, then equals to SUPPORTS_AVAILABILITY of the connected plugin.'
+      creationDate:
+        type: string
+        description: 'The date the activity was first created.'
+      lastModified:
+        type: string
+        description: 'The date that the activity content was last modified.'
+      lastPublished:
+        type: string
+        description: 'The date that the activity was last changed and published.'
+      published:
+        type: boolean
+#        description: 'NEED MORE INFO!, as could not find when it is true. Unpublished product is not returned in response.'
+      title:
+        type: string
+        description: 'The title of the activity.'
+      description:
+        type: string
+        description: 'Text describing this activity. May contain HTML markup.'
+      excerpt:
+        type: string
+        description: 'A short, non-HTML summary (max 255 chars) describing the product. This text may be used in activity preview pages.'
+      cancellationPolicy:
+        $ref: '#/definitions/CancellationPolicyDto'
+        description: 'The cancellation policy, describing any cancellation fees incurred when cancelling bookings.'
+      overrideBarcodeFormat:
+        type: boolean
+#        description: 'NEED MORE INFORMATION: probably does not work. Specifies if ticket barcode format for this activity overrides standard QR_CODE.'
```

Comment:
This does work. There is barcode value per-vendor, and if product uses another barcode than that, this value is true. As well it has `barcodeType` set. Otherwise `barcodeType` is ignored.

Reply:
Yes, "overrideBarcodeFormat" itself is working, however its value is not sent in the API response.

Reply:
Recommendation: file as bug